seungjong/routes/userRouter.js

```diff
@@ -1,8 +1,8 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const userController = require('../controllers/userController');
+const userController = require("../controllers/userController");
 
+router.post("/signup", userController.signUp);
+router.get("/logIn", userController.logIn);
```

Comment:
```suggestion
router.post("/logIn", userController.logIn);
```
login 정보를 받을 때는 body를 통해 받으므로 post 메소드를 통해 처리되어야합니다.